saleor/plugins/manager.py

```diff
@@ -70,15 +70,12 @@ def __run_method_on_plugins(
         self, method_name: str, default_value: Any, *args, **kwargs
     ):
         """Try to run a method with the given name on each declared plugin."""
-        with opentracing.global_tracer().start_active_span(
-            f"PluginsManager.{method_name}"
-        ):
-            value = default_value
-            for plugin in self.plugins:
-                value = self.__run_method_on_single_plugin(
-                    plugin, method_name, value, *args, **kwargs
-                )
-            return value
+        value = default_value
```

Comment:
I think that we could consider leaving the span here. It will produce one an additional call for each manager call, but we will see the total amount of time consumed by plugins. Not sure if this is something important for us.